Documentation/contributing/development/images.rst

```diff
@@ -32,19 +32,84 @@ You can then push the image tag to your own registry for development builds:
 Official release images
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Anyone can build official release images using the make target below but
-pushing to the official registries is restricted to Cilium maintainers. Ask in
-the #launchpad Slack channels for the exact details.
+Anyone can build official release images using the make target below.
 
 ::
 
     DOCKER_IMAGE_TAG=v1.4.0 make docker-image
 
-You can then push the image tag to the registry:
+Official Cilium repositories
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following table contains the main container image repositories managed by
+Cilium team. It is planned to convert the build process for all images based
+on GH actions.
+
++-------------------------------+---------------------------------------------+------------------------------------------+-------------------------+-------------------+
+|     **Github Repository**     |                **Dockerfile**               |      **container image repository**      |   **Architectures**     | **Build process** |
+|                               |                                             |                                          +-----------+-------------+                   |
+|                               |                                             |                                          | **amd64** | **aarch64** |                   |
++-------------------------------+---------------------------------------------+------------------------------------------+-----------+-------------+-------------------+
+| github.com/cilium/cilium      | contrib/packaging/docker/Dockerfile.runtime | quay.io/cilium/cilium-runtime            |     Y     |      N      |     Quay auto     |
```

Comment:
Nit: Might be worth removing `github.com` to gain a little space here. `contrib/packaging/docker` is also making the second column longer than necessary. It's not going to make the table fit the page though, so probably not worth re-pushing.